### PR TITLE
perf(test): improve timeout for test cases

### PR DESCRIPTION
**Pull request by shreyas4510**
Timeout values added for test cases from environment variables.

Fixes issue #100.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/eicrud/eicrud/blob/develop/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (optional)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
- [ ] Yes
- [ ] No

### CODE REVIEW

File: .github/workflows/action-publish.yml
[the remark this answers is not in the document]
Comment (acrosett) [reply .github/workflows/action-publish.yml:1]: this seems to be in the wrong place, here it is attached to the mongo service

File: .github/workflows/action-test.yml
Comment (acrosett) [inline .github/workflows/action-test.yml:1]: same here

File: test/core/core.account.spec.ts
Comment (acrosett) [inline test/core/core.account.spec.ts:1]: some tests that were 7000 are now 6000 with the new default, it can break the CI if these are lowered

**Review by acrosett — CHANGES_REQUESTED** [review]
added a few more comments,

ideally we could make use of the `setup.js` mentioned in this [issue](https://stackoverflow.com/questions/58042401/configure-jest-timeout-once-for-all-tests) so that we can set the default timeout for all tests and ditch the `"testTimeout"` in `package.json`. However we can already merge this if you want

File: test/src/app.module.ts
Comment (acrosett) [inline test/src/app.module.ts:1]: small typo here, `postgresPasword` should be `postgresPassword`

File: package.json
Comment (acrosett) [inline package.json:1]: we need to keep this, the default timeout for tests is 5000 so if you remove this it will reduce and might break the CI

**Review by acrosett — CHANGES_REQUESTED** [review]

File: package.json
Comment (shreyas4510) [reply package.json:1]: No I have updated default to 8000 in env.ts file so when no env variable which will be true in case of CI then it will take 8000

Comment (acrosett) [reply package.json:1]: my bad, didn't see the `jest.setTimeout(timeout);`, all good

**Review by acrosett — APPROVED** [review]

**Pull request merged**